roles/openshift_facts/library/openshift_facts.py

```diff
@@ -875,7 +875,7 @@ def set_version_facts_if_unset(facts):
                 version_gte_3_3_or_1_3 = version >= LooseVersion('1.3.0')
                 version_gte_3_4_or_1_4 = version >= LooseVersion('1.4.0')
                 version_gte_3_5_or_1_5 = version >= LooseVersion('1.5.0')
-                version_gte_3_6_or_1_6 = version >= LooseVersion('1.6.0')
+                version_gte_3_6_or_1_6 = version >= LooseVersion('3.6.0') || version >= LooseVersion('1.6.0')
```

Comment:
With the unified versioning, we don't even really need the version_gte_* variables. As long as we can access the version, we can move the comparisons to where they are used with the `version_compare` filter: https://docs.ansible.com/ansible/playbooks_tests.html#version-comparison